### PR TITLE
Proposal - AzureFunctionApps (Independent Publisher)

### PR DESCRIPTION
**Pull request by Stephan-Rijk**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [x] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [x] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [x] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [x] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

**Comment by Stephan-Rijk** [comment]
Note! We don't want to publish this custom connector, but we want to use it privately. We just want it certified, so we can use the custom connector in our solution.

File: independent-publisher-connectors/AzureFunctionApps/Readme.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Azure Function Apps Connector for Application Registrations
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/Readme.md:1]: @Stephan-Rijk please include all required sections in the readme file. please follow our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for raw code view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
+{
+    "swagger": "2.0",
+    "info": {
+        "title": "Default title",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:4]: @Stephan-Rijk Please give a proper title for the connector

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        "description": "",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:5]: @Stephan-Rijk please define a proper description for the connector.

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        "version": "1.0"
+    },
+    "host": "[YOURFUNCTIONAPP].azurewebsites.net",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:8]: @Stephan-Rijk to use a dynamic host please setup the apiProperties.json file properly. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/policy-templates/dynamichosturl/dynamichosturl).

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    "basePath": "/",
+    "schemes": [
+        "https"
+    ],
+    "consumes": [],
+    "produces": [],
+    "paths": {
+        "/api/DynamicsAuthStart": {},
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:16]: @Stephan-Rijk can you please explain the empty paths? is it possible to define them or remove them?

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+        "/": {},
+        "/DynamicsAuth": {},
+        "/api/DynamicsAuth": {
+            "post": {
+                "responses": {
+                    "default": {
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:22]: @Stephan-Rijk it is not allowed to have a default response only. please define a proper response. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+                        "description": "default",
+                        "schema": {}
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:24]: @Stephan-Rijk empty schema is not allowed. please define a proper schema.

File: independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+                    }
+                },
+                "summary": "DynamicsAuth",
+                "operationId": "DynamicsAuth",
+                "description": "DynamicsAuth",
```
Comment (Amjed-Ayoub) [inline independent-publisher-connectors/AzureFunctionApps/apiDefenition.swagger.json:29]: @Stephan-Rijk I would highly recommend you having different values for summary and description. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@Stephan-Rijk you are missing the apiProperties.json file. please see our template [here](https://github.com/microsoft/PowerPlatformConnectors/tree/dev/templates/Independent%20Publisher).

**Comment by Stephan-Rijk** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Comment by vmanoharas** [comment]
Hello @Stephan-Rijk, 

The comments are waiting to be resolved from 4 months, please resolve them.

**Comment by vmanoharas** [comment]
Hello @Stephan-Rijk, 

This PR is in open state from many months. I am closing this PR, kindly reopen/create new PR, if you have any progress.

**Pull request closed without merging**